core/store/migrate/migrations/0090_ocr2_relay.sql

```diff
@@ -0,0 +1,28 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE offchainreporting2_oracle_specs
+    ADD COLUMN relay text,
+    ADD COLUMN relay_config JSONB,
+    DROP COLUMN evm_chain_id,
```

Comment:
What about migrating existing? I guess since this hasn't been released yet, we don't have anything in the wild

Reply:
yeah exactly, don't think its worth the trouble